reposerver/repository/repository.proto

```diff
@@ -15,19 +15,21 @@ message ManifestRequest {
     // Name of the application for which the request is triggered
     string appName = 5;
     string namespace = 8;
-    github.com.argoproj.argo_cd.v2.pkg.apis.application.v1alpha1.ApplicationSource applicationSource = 10;
```

Comment:
Just noticed that field numbers had changed while we only need to add `hasMultipleSources` and `refSources`. Is it really necessary? 

If not can you please revert field number changes? Otherwise upgrade will be painful (controller wont be able to talk to repo server during upgrade, self managed argocd won't be able to finish upgrade )